translation-service-google/src/main/java/eu/europeana/api/translation/service/google/GLangDetectConstants.java

```diff
@@ -0,0 +1,43 @@
+package eu.europeana.api.translation.service.google;
+
+import java.util.List;
+
+import eu.europeana.api.translation.service.util.ThresholdByTextLength;
+
+/**
+ * Constants used by the Google language detection service
+ * 
+ * @author Nuno Freire
+ * @since 29/01/2025
+ */
+public class GLangDetectConstants {
+
+	protected static final List<ThresholdByTextLength> thresholdsForEdmLanguageHint = List.of(
```

Comment:
Please consider to shorten the function names without losing their clarity, e.g. here thresholdEdmLangHint, or down thresholdNoHintVeryHighPrec.